app/views/admin/settings/index.html.erb

```diff
@@ -86,8 +86,19 @@
         </p>
       </div>
       <div class="settings-section email hidden" data-hook="admin_settings_email">
-        <%= f.select 'email.mail_service', options_for_select([['Mailgun', 'mailgun'],['Sendgrid', 'sendgrid'],['Mandrill','mandrill'],['Postmark','postmark'],['Sparkpost','sparkpost']], AppSettings['email.mail_service']), label: "Inbound Mail Service Provider" %>
-
+        <%= f.select 'email.mail_service', options_for_select([['Mailgun', 'mailgun'],['Sendgrid', 'sendgrid'],['Mandrill','mandrill'],['Postmark','postmark'],['Sparkpost','sparkpost'],['POP3','pop3'], ['IMAP','imap']], AppSettings['email.mail_service']), label: "Inbound Mail Service Provider" %>
+        <div class="pop3-settings <%= "hidden" if "#{AppSettings['email.mail_service']}" != 'pop3' %>">
+          <p>POP3 will require run 'rake helpy:mailman'. Also changes will require a restart of the rake task</p>
+          <%= f.text_field 'email.pop3_server', value: AppSettings['email.pop3_server'], label: "POP3 Host" %>
+          <%= f.text_field 'email.pop3_username', value: AppSettings['email.pop3_username'], label: "POP3 Login" %>
+          <%= f.password_field 'email.pop3_password', value: AppSettings['email.pop3_password'], label: "POP3 Password" %>
+        </div>
+        <div class="imap-settings <%= "hidden" if "#{AppSettings['email.mail_service']}" != 'imap' %>">
+          <p>IMAP will require run 'rake helpy:mailman'. Also changes will require a restart of the rake task</p>
+          <%= f.text_field 'email.imap_server', value: AppSettings['email.imap_server'], label: "IMAP Host" %>
+          <%= f.text_field 'email.imap_username', value: AppSettings['email.imap_username'], label: "IMAP Login" %>
+          <%= f.password_field 'email.imap_password', value: AppSettings['email.imap_password'], label: "IMAP Password" %>
+        </div>
         <%= f.form_group 'email.send_email', label: { text: "Enable Outbound (SMTP) Email" }, class: 'send-email-toggle' do %>
```

Comment:
Sure. So toggle would set the SSL Flag. What would be the variable? And should SSL be on by default?